README.md

```diff
@@ -2289,29 +2289,69 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
-* <a id='switch-never-default'></a>(<a href='#switch-never-default'>link</a>) **Never use the `default` case when `switch`ing over an enum.**
+* <a id='switch-never-default'></a>(<a href='#switch-never-default'>link</a>) When switching over an enum, generally prefer enumerating all cases rather than using the `default` case.
```

Comment:
@calda I'm not sure what the right thing to do here is but do you think we should change the slug? 

```suggestion
* <a id='switch-avoid-default'></a>(<a href='#switch-avoid-default'>link</a>) When switching over an enum, generally prefer enumerating all cases rather than using the `default` case.
```

This change would break any links in the wild to the old wording. This may be what we want in this case, since we are intentionally softening this rule.

Reply:
In some ways I think it would be ideal to have "prefer" in the new slug but I didn't think of a good way to do that.

Reply:
I think `switch-avoid-default` is good. I also think changing the slug is a good idea, thanks for the suggestion.